File: AtividadesEscola/atv01PHP/qst04.php
<?php

while (true){
    echo "DIgite a hora atual: ";
    $hora = intval(readline());

    if ($hora >= 5 && $hora <= 11) {
        echo "Bom dia!\n";
    } elseif ($hora >= 12 && $hora <= 18) {
        echo "Boa tarde!\n";
    } elseif ($hora >= 19 && $hora <= 23) {
        echo "Boa noite!\n";
    } elseif ($hora <= 4 || $hora >= 24) { 
        echo "Vai dormir, seu fela!\n";
    }
}

?>

File: AtividadesEscola/atv01Python/qst04.py
while(True):
    hora = int(input("Digite a hora atual: "))

    if hora >= 5 and hora <= 11:
        print("Bom dia!")
    elif hora >= 12 and hora <= 18:
        print("Boa tarde!")
    elif hora >= 19 and hora <= 23:
        print("Boa noite!")
    elif hora <= 4 or hora >= 24:  
        print("Vai dormir, seu fela!") 